src/sparseml/pytorch/utils/exporter.py

```diff
@@ -154,13 +238,38 @@ def export_onnx(
             _delete_trivial_onnx_adds(onnx_model)
             onnx.save(onnx_model, onnx_path)
 
+    def export_torchscript(
+        self,
+        name: str = "model.pts",
+        trace: bool = False,
+        sample_batch: Optional[Any] = None,
```

Comment:
nit: can we remove `trace`, and just run a trace if a `sample_batch` is provided?